File: webui/lmApp/src/app/model/Menu.ts
export class Menu {

  constructor(private _MenuId:number, private _MenItem:String,private _MenCalories:number,private _VenId:number,
      private _Ratings:number, private _Reviews:String, private _MenPrice:number, private _OrdId:number,
      private _OrderItemId:number, private _Qty:number, private _VenName:String, private _MenSpeciality:String) {
  }
          set MenuId(MenuId:number){
           this._MenuId = MenuId;
          }
  
         get MenuId():number{
          return this._MenuId;
         }
         set MenSpeciality(MenSpeciality:String){
          this._MenSpeciality= MenSpeciality;
         }
  
        get MenSpeciality():String{
         return this._MenSpeciality;
        }
         set OrdId(OrdId:number){
          this._OrdId = OrdId;
         }
  
        get OrdId():number{
         return this._OrdId;
        }
        set MenItem(MenItem:String){
          this._MenItem= MenItem;
         }
  
        get MenItem():String{
         return this._MenItem;
        }
  
        set Reviews(Reviews:String){
          this._Reviews= Reviews;
         }
  
        get Reviews():String{
         return this._Reviews;
        }
  
        set OrdItemId(OrderItemId:number){
          this._OrderItemId = OrderItemId;
         }
  
        get OrdItemId():number{
         return this._OrderItemId;
        }
        set Qty(Qty:number){
          this._Qty = Qty;
         }
  
        get Qty():number{
         return this._Qty;
        }
  
         set MenCalories(MenCalories:number){
          this._MenCalories = MenCalories;
         }
  
         get MenCalories():number{
          return this._MenCalories;
         }
  
         set VenId(VenId:number){
          this._VenId = VenId;
         }
  
        get VenName():String{
         return this._VenName;
        }
        set VenName(VenName:String){
          this._VenName = VenName;
         }
  
        get VenId():number{
         return this._VenId;
        }
  
        set Ratings(Ratings:number){
          this._Ratings = Ratings;
         }
  
        get Ratings():number{
         return this._Ratings;
        }
  
  
        set MenPrice(MenPrice:number){
          this._MenPrice = MenPrice;
         }
  
        get MenPrice():number{
         return this._MenPrice;
        }
  
  }

File: restservice/canteenmanagement/src/main/java/com/hexaware/FTP112/model/Menu.java
package com.hexaware.FTP112.model;

import java.util.Objects;

/**
 * Menu class used to display menu information.
 * @author hexaware
 */
public class Menu {
/**
 * foodId to store menuId.
 */
  private int menuId;
  private String menItem;
  private double menCalories;
  private int venId;
  private int ratings;
  private String reviews;
  private String venName;
  private String venPhnNo;
  private double menPrice;
  private String menSpeciality;
  /**
   * Default Constructor.
   */
  public Menu() {

  }
/**
 * @param argMenuId to initialize menu id.
 * @param argMenItem to initialize menu item.
 * @param argMenPrice to initialize menu price.
 * @param argMenCalories to initialize calories.
 * @param argVenId to initialize vendor id.
 * @param argRatings to initialize vendor rating.
* @param argReviews to initialize vendor reviews.
 * used to get details through constructor.
 */
  public Menu(final int argMenuId, final String argMenItem,
              final double argMenCalories,
              final int argVenId, final int argRatings, final String argReviews, final double argMenPrice) {
    this.menuId = argMenuId;
    this.menItem = argMenItem;
    this.menPrice = argMenPrice;
    this.menCalories = argMenCalories;
    this.venId = argVenId;
    this.ratings = argRatings;
    this.reviews = argReviews;

  }
  @Override
    public final boolean equals(final Object obj) {
    if (obj == null) {
      return false;
    }
    if (getClass() != obj.getClass()) {
      return false;
    }
    Menu menu = (Menu) obj;
    if (Objects.equals(menuId, menu.getMenId())
        && Objects.equals(menItem, menu.getMenItem())
        && Objects.equals(menPrice, menu.getMenPrice())
        && Objects.equals(menCalories, menu.getMenCalories())
        && Objects.equals(venId, menu.getVenId())
        && Objects.equals(ratings, menu.getRatings())
        && Objects.equals(reviews, menu.getReviews())) {
      return true;
    }
    return false;
  }
  @Override
    public final int hashCode() {
    return Objects.hash(menuId, menItem, menPrice, menCalories, venId, ratings, reviews);
  }
    /**
     * @return this menId.
     */
  public final int getMenId() {
    return this.menuId;
  }
    /**
     * @return this food type.
     */
  public final String getMenSpeciality() {
    return this.menSpeciality;
  }
  /**
     * @return this Men item.
     */
  public final String getMenItem() {
    return this.menItem;
  }
  /**
     * @return this Menu Price.
     */
  public final Double getMenPrice() {
    return this.menPrice;
  }
  /**
     * @return this calories .
     */
  public final double getMenCalories() {
    return this.menCalories;
  }
  /**
     * @return this ven id .
     */
  public final int getVenId() {
    return this.venId;
  }
  /**
     * @return this ratings .
     */

  public final int getRatings() {
    return this.ratings;
  }
  /**
     * @return this reviews .
     */
  public final String getReviews() {
    return this.reviews;
  }
  /**
     * @return this ven name .
     */
  public final String getVenName() {
    return this.venName;
  }
  /**
     * @return this ven phone number .
     */
  public final String getVenPhnNo() {
    return this.venPhnNo;
  }
    /**
     * @param argMenSpeciality gets the food type.
     */
  public final void setMenSpeciality(final String argMenSpeciality) {
    this.menSpeciality = argMenSpeciality;
  }
    /**
     * @param argMenId gets the food id.
     */
  public final void setMenId(final int argMenId) {
    this.menuId = argMenId;
  }
    /**
     * @param argMenItem gets the Menu item.
     */
  public final void setMenItem(final String argMenItem) {
    this.menItem = argMenItem;
  }
    /**
     * @param argMenPrice gets the Menu Price.
     */
  public final void setMenPrice(final double argMenPrice) {
    this.menPrice = argMenPrice;
  }
    /**
     * @param argMenCalories gets the calories.
     */
  public final void setMenCalories(final double argMenCalories) {
    this.menCalories = argMenCalories;
  }
    /**
     * @param argVenName gets the calories.
     */
  public final void setVenName(final String argVenName) {
    this.venName = argVenName;
  }
    /**
     * @param argVenPhnNo gets the calories.
     */
  public final void setVenPhnNo(final String argVenPhnNo) {
    this.venPhnNo = argVenPhnNo;
  }
}
